src/status_im/subs.cljs

```diff
@@ -2930,3 +2930,9 @@
            (>= (count new-password) 6)
            (>= (count current-password) 6)
            (= new-password confirm-new-password))})))
+
+(re-frame/reg-sub
+  :bookmarks/active
+  :<- [:bookmarks]
+  (fn [bookmarks]
+    (filter #(= false (get (second %1) :removed false)) bookmarks)))
```

Comment:
if you have `false` as default value , why do you need to compare with false? it could be just `(filter #(get (second %1) :removed false) bookmarks)`

Reply:
or if default `false` you don't need it, because nil will work as false
`(remove #(:removed (second %)) bookmarks)`

Reply:
modified